python/hail/keytable.py

```diff
@@ -9,6 +9,40 @@
 class KeyTable(object):
     """Hail's version of a SQL table where columns can be designated as keys.
 
+    KeyTables can be imported from a text file with :py:meth:`~hail.HailContext.import_keytable` or generated from an existing
+    VariantDataset with :py:meth:`~hail.VariantDataset.aggregate_by_key`, :py:meth:`~hail.VariantDataset.make_keytable`,
+    :py:meth:`~hail.VariantDataset.samples_keytable`, and :py:meth:`~hail.VariantDataset.variants_keytable`.
+
+    In the examples below, we have imported two KeyTables from text files (``kt1`` and ``kt2``).
+
+    >>> kt1 = hc.import_keytable("data/kt_example1.tsv", ["ID"], config=TextTableConfig(impute=True))
+
+    +--+---+---+-+-+----+----+----+
```

Comment:
Not a comment on this PR, but we should make a show() method on keytables like dataframes.  or just make it easy to do that with a dataframe.